### DISCUSSION TITLE
User extend field want pass in cli creating user

### DISCUSSION THREAD

**Discussion by dvrg** [pallets-eco/flask-security#588] [Q&A] [answered]
Hi, i had extend User with add first and last name

![image](https://user-images.githubusercontent.com/15957000/159120419-b63d44f3-5951-4ed2-a4e7-1677aa5fb687.png)

when i create new user with cli, cli want first and last name

![image](https://user-images.githubusercontent.com/15957000/159120454-52c19cd1-7639-4ed4-8446-cd7a29cc931f.png)

any suggestion how i pass the first and last name when creating user from cli ? Thanks in advance

**Comment by jwag956** [comment, marked as answer]
If you do flask users create --help it will show you the syntax. Additional attributes can be set using the attr:value syntax:

first_name:Chris

**Reply by dvrg** [reply]
yes, it's works. 
as simple as that but i'm missing. Thankyou
